Agreements/Internal/S29_7.txt

```diff
@@ -1 +1,4 @@
-COPX:EGNM:*:PENIL:*:*:Sector 29:London S7:28000:*:PENIL
\ No newline at end of file
+COPX:EGNM:*:PENIL:*:*:Sector 29:London S7:28000:*:PENIL
+
+COPX:NOKIN:*:*:*:*:Sector 29:London S7:28000:*:^NOKIN
+COPX:REXAM:*:*:*:*:Sector 29:London S7:28000:*:^REXAM
```

Comment:
I fear this will also try to be applied to aircraft at cruise via NOKIN/REXAM? Could you test please @AleksMax?